installer/bundle/bundle_skel_AIX.sh

```diff
@@ -0,0 +1,432 @@
+            local shouldexit=true
+            shift 2
+            ;;
+
+        --extract-binary)
+            # hidden option, not part of usage
+            # echo "  --extract-binary FILE  extract the binary to FILE."
+            tail -n +${SCRIPT_LEN_PLUS_ONE} "${SCRIPT}" > "$2"
+            local shouldexit=true
+            shift 2
+            ;;
+
+        --extract)
+            verifyNoInstallationOption
+            installMode=E
+            shift 1
+            ;;
+
+        --force)
+            forceFlag=true
+            shift 1
+            ;;
+
+        --install)
+            verifyNoInstallationOption
+            installMode=I
+            shift 1
+            ;;
+
+        --purge)
+            verifyNoInstallationOption
+            installMode=P
+            shouldexit=true
+            shift 1
+            ;;
+
+        --remove)
+            verifyNoInstallationOption
+            installMode=R
+            shouldexit=true
+            shift 1
+            ;;
+
+        --restart-deps)
+            restartDependencies=--restart-deps
+            shift 1
+            ;;
+
+        --source-references)
+            source_references
+            cleanup_and_exit 0
+            ;;
+
+        --upgrade)
+            verifyNoInstallationOption
+            installMode=U
+            shift 1
+            ;;
+
+        --version)
+            echo "Version: `getVersionNumber $OM_PKG scx-`"
+            exit 0
+            ;;
+
+        --debug)
+            echo "Starting shell debug mode." >&2
+            echo "" >&2
+            echo "SCRIPT_INDIRECT: $SCRIPT_INDIRECT" >&2
+            echo "SCRIPT_DIR:      $SCRIPT_DIR" >&2
+            echo "EXTRACT DIR:     $EXTRACT_DIR" >&2
+            echo "SCRIPT:          $SCRIPT" >&2
+            echo >&2
+            set -x
+            shift 1
+            ;;
+
+        -? | --help)
+            usage `basename $0` >&2
+            cleanup_and_exit 0
+            ;;
+
+        *)
+            usage `basename $0` >&2
+            cleanup_and_exit 1
+            ;;
+    esac
+done
+
+if [ -z "${installMode}" ]; then
+    echo "$0: No options specified, specify --help for help" >&2
+    cleanup_and_exit 3
+fi
+
+#
+# Note: From this point, we're in a temporary directory. This aids in cleanup
+# from bundled packages in our package (we just remove the diretory when done).
+#
+
+mkdir -p $EXTRACT_DIR
+cd $EXTRACT_DIR
+
+# Do we need to remove the package?
+if [ "$installMode" = "R" -o "$installMode" = "P" ]
+then
+    if [ -f /opt/microsoft/scx/bin/uninstall ]; then
+        /opt/microsoft/scx/bin/uninstall $installMode
+    else
+        # This is an old kit.  Let's remove each separate provider package
+        for i in /opt/microsoft/*-cimprov; do
+            PKG_NAME=`basename $i`
+            if [ "$PKG_NAME" != "*-cimprov" ]; then
+                echo "Removing ${PKG_NAME} ..."
+                pkg_rm ${PKG_NAME}
+            fi
+        done
```

Comment:
Lines 280 - 286 is to remove bundled packages. Since UNIX doesn't have that, you can remove this in all UNIX-based installation scripts. For UNIX, just removing scx and OMI (done at 289-290) should be fine.